string_unique/string-app/src/main/java/net/mtnboy/StringUnique/App.java

```diff
@@ -10,6 +10,10 @@ public static void main( String[] args ) {
 
     //static since this doesn't require storage outside of this one invocation.
     public static boolean hasAllUnique(String str) {
+        if ((str == null) || (str.isEmpty())) {
+            return true;
```

Comment:
Is a string unique if it doesn't exist? I guess it technically has a unique set of characters since that set is empty?

Reply:
A boundary condition yes. But since hasAllUnique can be defined as if there are 2 or more identical characters, then it is not unique, otherwise it is means null and empty are unique.

but consider it as boundary I think...